vscode/oso/syntaxes/polar.tmLanguage.json

```
@@ -80,7 +80,7 @@
     },
     "resource-block": {
       "name": "meta.resource-block",
-      "begin": "(resource|actor)\\s*([a-zA-Z][a-zA-Z0-9:]*)\\s*\\{",
+      "begin": "(resource|actor)\\s*([a-zA-Z][a-zA-Z0-9:_]*)\\s*\\{",
```

Gonna sneak a couple more improvements in while we're at it. Changes:
- Enforce at least one space between `resource`/`actor` and the identifier. Invalid: `actorUser{}`; valid: `actor User{}`.
- Identifiers can start with an underscore.
- Colons in an identifier must come in pairs *and* must not come at the end of the identifier.
```suggestion
      "begin": "(resource|actor)\\s+([a-zA-Z_][a-zA-Z0-9_]*(::[a-zA-Z0-9_]+)*)\\s*\\{",
```